m2m-transfers/m2m-transfers-json-contract.md

```diff
@@ -134,54 +131,99 @@ list of institution and member limits, eligible shares, list of scheduled transf
   "loggingErrorMessage": "No eligible shares found"
 }
 ```
-## PROCESS REQUEST STATE
-### Client Request (PROCESS)
+## CREATE TRANSFER REQUEST STATE
+### Client Request (CREATETRAN)
 
 ```json
 {
-  "rgState": "PROCESS",
+  "rgState": "CREATETRAN",
   "powerOnFileName": "BANNO.M2MTRANSFERS.V1.POW",
   "userChrList": [
-    { "id": 1, "value": "[request type]:[data]|[data]|[data]|[data]|[data]|[data]" },
-    { "id": 2, "value": "[data]|[data]|[data]|[data]|[data]|[data]" },
-    { "id": 3, "value": "" },
-    { "id": 4, "value": "" },
+    { "id": 1, "value": "395" }, // recipientLoc if available, empty if not
+    { "id": 2, "value": "1234567890S0001|9876543210L0001|weekly|12/31/2021|1|15" },  // [sourceAccount][destinationAccount][frequency]|[startDate]|[day1]|[day2]
+    { "id": 3, "value": "HUB|nickname" }, // [first 3][nickname]
+    { "id": 4, "value": "internal memo for immediate transfers" }, // internal memo for immediate transfers (max 132 characters)
     { "id": 5, "value": "" }
   ],
-  "userNumList": [], //  usernum[1-5] is unused
+  "userNumList": [
+	{ "id": 1, "value": 1000.51 } // transferAmt
```

Comment:
Storing a monetary value in a userNumList variable would be done without the decimal (decimal assumed). (here and other areas in the contract)